<launch>
    <node name="tracker_node" pkg="icog_face_tracker" type="tracker_node" output="screen">
        <param name="show_preview" value="true"/>
        <param name="camera_topic" value="/pan_and_tilt/main_cam/image_raw"/>
    </node>

    <node name="id_no_send" pkg="icog_face_tracker" type="id_no_send" output="screen">
        
        <param name="camera_topic_2" value="/main_cam_2/camera_topic_2"/>
    </node>
</launch>
